testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js

```diff
@@ -1021,7 +1021,16 @@ QUnit.module('Lookup', {
         mouse.start().down().move(0, 10).up();
 
         openPopupWithList(firstLookup);
-        assert.equal($('.' + LIST_ITEM_SELECTED_CLASS).length, 2);
+
+        // NOTE: in ShadowDOM mode one selected item is inside ShadowDOM
```

Comment:
why so strange behavior?

Reply:
Opened lookup is in overlaym while overlays work in document not in shadowDom, while closed popup still is in shadow dom.